internal/exec/aws_utils.go

```diff
@@ -0,0 +1,73 @@
+package exec
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials/stscreds"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	log "github.com/charmbracelet/log"
+
+	errUtils "github.com/cloudposse/atmos/errors"
+)
+
+// loadAWSConfig loads AWS config.
+/*
+	It looks for credentials in the following order:
```

Comment:
I don't know why we are writing this as a custom implementation. This is exactly what the SDK does. We should not implement this.